include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -10624,6 +10624,11 @@
           "enumerant" : "Slang",
           "value" : 11,
           "version" : "1.0"
+        },
+        {
+          "enumerant" : "Zig",
+          "value" : 12,
+          "version" : "1.0",
```

Comment:
One of the builds complains about the next line. I assume because of this trailing comma.